diff -dur AMB-plugins-0.8.1.orig/Makefile AMB-plugins-0.8.1/Makefile
--- AMB-plugins-0.8.1.orig/Makefile	2010-11-14 23:46:42.000000000 +0100
+++ AMB-plugins-0.8.1/Makefile	2016-10-10 20:16:21.527284513 +0200
@@ -14,7 +14,8 @@
 #    along with this program; if not, write to the Free Software
 #    Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 
-
+CXX = g++
+LDFLAGS =
 CPPFLAGS += -I. -fPIC -D_REENTRANT -Wall -O3
 
 
@@ -22,28 +23,28 @@
 
 
 ambisonic0.so:	ambisonic0.o ambisonic0_if.o virtmic.o allpass.o
-	g++ -shared -o $@ ambisonic0.o ambisonic0_if.o virtmic.o allpass.o
+	$(CXX) -shared $(LDFLAGS) -o $@ ambisonic0.o ambisonic0_if.o virtmic.o allpass.o
 
 ambisonic0.o:	ladspaplugin.h ambisonic0.h
 ambisonic0_if.o:	ladspaplugin.h ambisonic0.h
 
 
 ambisonic1.so:	ambisonic1.o ambisonic1_if.o filter1.o
-	g++ -shared -o $@ ambisonic1.o ambisonic1_if.o filter1.o
+	$(CXX) -shared $(LDFLAGS) -o $@ ambisonic1.o ambisonic1_if.o filter1.o
 
 ambisonic1.o:	ladspaplugin.h ambisonic1.h filter1.h
 ambisonic1_if.o:	ladspaplugin.h ambisonic1.h
 
 
 ambisonic2.so:	ambisonic2.o ambisonic2_if.o 
-	g++ -shared -o $@ ambisonic2.o ambisonic2_if.o
+	$(CXX) -shared $(LDFLAGS) -o $@ ambisonic2.o ambisonic2_if.o
 
 ambisonic2.o:	ladspaplugin.h ambisonic2.h
 ambisonic2_if.o:	ladspaplugin.h ambisonic2.h
 
 
 ambisonic3.so:	ambisonic3.o ambisonic3_if.o 
-	g++ -shared -o $@ ambisonic3.o ambisonic3_if.o
+	$(CXX) -shared $(LDFLAGS) -o $@ ambisonic3.o ambisonic3_if.o
 
 ambisonic3.o:	ladspaplugin.h ambisonic3.h
 ambisonic3_if.o:	ladspaplugin.h ambisonic3.h
